tools/build.js

```diff
@@ -0,0 +1,8 @@
+const fs = require('fs')
+const babel = require('@babel/core')
+
+const transpiled = babel.transformFileSync('./src/Headroom.js').code
+
+fs.writeFileSync('./index.js', transpiled)
```

Comment:
What ES version is output here? If this is ES6 we could have trouble later on.
This should be ES5 (with ES6 module syntax.)

Reply:
pretty sure its not es6 module syntax, but pretty sure that its es5^^